eppo/src/main/java/cloud/eppo/android/dto/TargetingRule.java

```diff
@@ -13,6 +13,10 @@ public String getAllocationKey() {
         return allocationKey;
     }
 
+    public void setAllocationKey(String allocationKey) {
```

Comment:
Why not cleanup `SerializedName` here too?

Reply:
🤦 Yes, we absolutely should! Good catch!